src/utils/layout-helpers.ts

```diff
@@ -0,0 +1,387 @@
+import * as walkSync from 'walk-sync';
+import { join, sep, extname, dirname } from 'path';
+import { CompletionItem, CompletionItemKind } from 'vscode-languageserver';
+
+import { readFileSync, existsSync } from 'fs';
+
+export function isMuApp(root: string) {
```

Comment:
Since module unification is [officially dead](https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html), I'm going to remove all the code for it from this. It should simplify things a bit and improve performance a bit as a result.